### PR TITLE
[CCTRI-1376] Add documentation

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,289 @@
...
+  - Value from `.data.report.blacklists.engines[].reference` will map to `source_uri`
+  - The query time will map to `.observed_time.start_time`
+
+- `Indicator` from each entry in `.data.report.blacklists.engines` where `.data.report.blacklists.engines[].detected` is `true`:
+  - `Indicator` TLP will be `white`
+  - Value from `.data.report.blacklists.engines[].engine` will map to `short_description`
+  
+- `Relationship` between `Sighting` and `Indicator` with the relationship type of `member-of` because they are based on feeds.
+
+
```
Comment (gdavoian) [inline README.md:288]: Redundant empty lines

File: README.md [hunk 2]
```diff
@@ -1 +1,289 @@
...
+  - Verifies the Authorization Bearer JWT and decodes it to restore the
+  original credentials.
+  - Makes a series of requests to the underlying external service to query for
+  some cyber threat intelligence data on each supported observable.
+  - Maps the fetched data into appropriate CTIM entities.
+  - Returns a list per each of the following CTIM entities (if any extracted):
+    - `Indicator`,
+    - `Sighting`,
+    - `Relationship`.
+
```
Comment (gdavoian) [inline README.md:249]: Redundant empty line

File: README.md (hunk 1 above)
Comment (AnastasiiaRozlyvan) [reply README.md]: Fixed, thank you!

File: README.md (hunk 2 above)
Comment (AnastasiiaRozlyvan) [reply README.md:249]: Done

**Pull request merged**